### DISCUSSION TITLE
my comfyui installation broke yesterday . 'execution_block_cb'line 313, in execute

### DISCUSSION THREAD

**Discussion by metamountain** [comfyanonymous/ComfyUI#4402] [Q&A] [answered]
Has anybody an idea how to fix it ?
The error occurs even with the  standard workflow....

Error occurred when executing CheckpointLoaderSimple:

map_node_over_list() got an unexpected keyword argument 'execution_block_cb'

  File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 313, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 188, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by metamountain** [comment]
Found the answer.  It was a broken custom node called ComfyUI_ezXY ...

**Reply by Maxadox** [reply]
how do I fix it?

**Reply by ltdrdata** [reply]
The author should patch.

**Reply by Maxadox** [reply]
How do I find it? I saw other people having same error in different nodes as well

**Reply by Maxadox** [reply]
https://discord.com/channels/1218270712402415686/1244085343905648691/1273674495336382576

**Comment by comfyanonymous** [comment, marked as answer]
https://github.com/comfyanonymous/ComfyUI/commit/86c5970ac030df5735bfa03495695e64ecc90984

This should prevent those nodes from breaking comfyui by hooking that function.
